litewallet/Views/DefaultCurrencyViewController.swift

```diff
@@ -22,11 +22,14 @@ class DefaultCurrencyViewController: UITableViewController, Subscriber {
 		didSet {
```

Comment:
The didSet block contains too much logic. It's better to extract this logic into helper functions to make the didSet block cleaner and more focused.

```
updateCurrencyRows(oldCurrencyCode: oldValue)
```